src/main/groovy/util/Event.groovy

```diff
@@ -0,0 +1,21 @@
+package groovy.util
+
+import groovy.transform.InheritConstructors;
+
+@InheritConstructors
+public class Event extends Closure
+{
+	private List<Closure> subscribers = []
```

Comment:
Seems reasonnable. As I said, I'm still very new in Groovy. This is the kind of differences I'm not aware of yet ;)